supporting_scripts/analysis-of-endpoint-connections/src/main/typeScript/tsconfig.analysisOfEndpointConnections.json

```diff
@@ -0,0 +1,11 @@
+{
+    "compilerOptions": {
+        "module": "CommonJS",
```

Comment:
I would prefer es2022 if possible

Reply:
When using es2022, executing the Script within the GitHub action fails. With CommonJS, that issue does not come up. That's why CommonJS is used here.